docs/core/install/linux-rhel.md

```diff
@@ -45,28 +45,51 @@ Consult the [Red Hat documentation for .NET](https://access.redhat.com/documenta
 
 ## RHEL 8 ✔️
 
-> [!TIP]
-> .NET 5.0 isn't yet available in the AppStream repositories, but .NET Core 3.1 is. To install .NET Core 3.1, use the `dnf install` command with the appropriate package, such as `aspnetcore-runtime-3.1` or `dotnet-sdk-3.1`. The following instructions are for .NET 5.0.
+.NET is included in the AppStream repositories for RHEL 8.
 
-[!INCLUDE [linux-prep-intro-generic](includes/linux-prep-intro-generic.md)]
+[!INCLUDE [linux-dnf-install-50](includes/linux-install-50-dnf.md)]
+
+## RHEL 7 ✔️ .NET 5.0
```

Comment:
This section is new. It's  a copy of the RHEL 7 .NET 3.1 section, adjusted for 5.0.